simulator/modules/fetch/fetch.cpp

```diff
@@ -188,6 +200,30 @@ void Fetch<FuncInstr>::clock( Cycle cycle)
 
     /* sending to decode */
     wp_datapath->write( std::move( instr), cycle);
+
+    if ( prefetch_enabled && !is_wrong_path)  // prefetch next line if enabled and wrong path isn't used
+    {
```

Comment:
Our code guideline is to use as less brackets as possible: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#pythonesque-statements